.github/workflows/build.yml

```diff
@@ -0,0 +1,195 @@
+          - TESTDIR: "JavaScript/es4x JavaScript/ringojs JavaScript/just"
+          - TESTLANG: "Julia"
+          - TESTLANG: "Kotlin"
+          - TESTLANG: "Lisp"
+          - TESTLANG: "Lua"
+          - TESTLANG: "Mumps"
+          - TESTLANG: "Nim"
+          - TESTLANG: "OCaml"
+          - TESTLANG: "Perl"
+          - TESTDIR: "PHP/php"
+          - TESTDIR: "PHP/comet PHP/kumbiaphp PHP/workerman PHP/webman"
+          - TESTDIR: "PHP/cakephp PHP/codeigniter PHP/fat-free PHP/fuel PHP/phpixie PHP/slim PHP/symfony PHP/yii2 PHP/zend PHP/spiral PHP/duckphp"
+          - TESTDIR: "PHP/amp PHP/hhvm PHP/peachpie PHP/php-ngx PHP/phalcon"
+          - TESTDIR: "PHP/hamlet PHP/laravel PHP/lumen PHP/swoole PHP/ubiquity PHP/hyperf PHP/sw-fw-less PHP/imi PHP/simps PHP/one"
+          - TESTLANG: "Prolog"
+          - TESTDIR: "Python/aiohttp Python/api_hour Python/apidaora Python/blacksheep Python/bottle Python/cherrypy Python/crax Python/django Python/emmett Python/eve Python/falcon Python/fastapi Python/flask"
+          - TESTDIR: "Python/hug Python/japronto Python/klein Python/morepath Python/pyramid Python/quart Python/responder Python/sanic Python/spyne Python/starlette"
+          - TESTDIR: "Python/tornado Python/turbogears Python/uvicorn Python/uwsgi Python/vibora Python/web2py Python/webware Python/weppy Python/wsgi"
+          - TESTDIR: "Ruby/agoo Ruby/grape Ruby/h2o_mruby Ruby/padrino Ruby/rack Ruby/rack-sequel"
+          - TESTDIR: "Ruby/rails Ruby/roda-sequel Ruby/sinatra Ruby/sinatra-sequel"
+          - TESTDIR: "Rust/actix Rust/gotham Rust/hyper Rust/iron Rust/saphir"
+          - TESTDIR: "Rust/may-minihttp Rust/nickel Rust/rocket"
+          - TESTDIR: "Rust/rouille Rust/thruster Rust/tokio-minihttp"
+          - TESTDIR: "Rust/warp-rust"
+          - TESTDIR: "Rust/roa"
+          - TESTDIR: "Rust/ntex"
+          - TESTDIR: "Scala/akka-http Scala/blaze Scala/cask Scala/colossus Scala/finagle"
+          - TESTDIR: "Scala/finatra Scala/finch Scala/http4s"
+          - TESTDIR: "Scala/play2-scala Scala/scalene Scala/youi"
+          - TESTDIR: "Scala/snunit"
+          - TESTDIR: "Scala/vertx-web-scala"
+          - TESTLANG: "Swift"
+          - TESTLANG: "TypeScript"
+          - TESTLANG: "Ur"
+          - TESTLANG: "V"
+          - TESTLANG: "Vala"
+          - TESTLANG: "VB"
+          - TESTLANG: "Mumps"
+      # Disable fail-fast to allow all failing frameworks/etc to fail in a
+      # single build, rather than stopping when the first one fails.
+      fail-fast: false
+    env:
+      TESTLANG: ${{ matrix.TESTLANG }}
+      TESTDIR: ${{ matrix.TESTDIR }}
+      COMMIT_MESSAGE: ${{ needs.setup.outputs.commit_message }}
+      BRANCH_NAME: ${{ needs.setup.outputs.branch_name }}
+      PR_NUMBER: ${{ github.event.pull_request.number }}
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 10
```

Comment:
Is there a reason for this specific number?

Reply:
Not sure. This was originally from when the setup job was still included in this one, so it's actually from the link in that job's comment: https://github.community/t/accessing-commit-message-in-pull-request-event/17158/2. I already tried removing the fetch-depth but it doesn't work without that, so I left it in.

TL;DR: It makes it work, so it's there. I could have investigated further but it was working and I didn't want to keep delaying the PR.

Reply:
Though I'm realizing now that the website it links to dynamically updates the URL by scroll so I didn't notice the URL changed before I copied it. The comment should link to https://github.community/t/accessing-commit-message-in-pull-request-event/17158/8. I'll fix that in a follow-up PR.